integration-tests/01_user.ts

```diff
@@ -37,8 +37,10 @@ export async function createNewPupil() {
                 lastname: "lastname:${userRandom}"
                 email: "test+${userRandom}@lern-fair.de"
                 newsletter: true
-                state: bw
                 registrationSource: normal
+                school: {
+                  state: bw
```

Comment:
What does just passing the state here do? It will not find a school and then do nothing? Or will it create a school for each pupil?

Reply:
This is the equivalent of having this new feature disabled OR users refusing to provide school information during registration, which is fine because it's optional.

So, In this case, we just fill the state field on the [pupil](https://github.com/corona-school/backend/blob/fd4c2578c219bd9721646a3e4f7af69926cf0f38/prisma/schema.prisma#L579) model since for schools we need the name as well.
I think the next step would be to move the school information we have on the pupil model to the school table...